src/qt/rpcconsole.cpp

```diff
@@ -1109,6 +1114,12 @@ void RPCConsole::updateDetailWidget()
         peerAddrDetails += "<br />" + tr("via %1").arg(QString::fromStdString(stats->nodeStats.addrLocal));
     ui->peerHeading->setText(peerAddrDetails);
     ui->peerServices->setText(GUIUtil::formatServicesStr(stats->nodeStats.nServices));
+    ui->peerRelayTxes->setText(stats->nodeStats.fRelayTxes ? "Yes" : "No");
```

Comment:
Yes, No, To, From not translatable?

I've launched bitcoin-qt in my language and I see some values translated (like `N/A` is translated to appropriate alternative), I assume these should be translated too.

Reply:
Those are new strings being introduced in the PR. They will need translations which will be done when they are placed on Transifex.

Reply:
I mean, these strings are not wrapped in `tr()`.

Reply:
Thanks @Talkless, done in #221.